src/sentry/rules/filters/assigned_to.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+
+from django import forms
+
+from sentry.rules.filters.base import EventFilter
+from sentry.mail.actions import MemberTeamForm
+
+
+class AssigneeTargetType:
+    NO_ONE = "NoOne"
+    TEAM = "Team"
+    MEMBER = "Member"
+
+
+CHOICES = [
+    (AssigneeTargetType.NO_ONE, "No One"),
+    (AssigneeTargetType.TEAM, "Team"),
+    (AssigneeTargetType.MEMBER, "Member"),
+]
+
+
+class AssignedToForm(MemberTeamForm):
+    targetType = forms.ChoiceField(choices=CHOICES)
+
+    teamValue = AssigneeTargetType.TEAM
+    memberValue = AssigneeTargetType.MEMBER
+
+
+class AssignedToFilter(EventFilter):
+    form_cls = AssignedToForm
+    label = "The issue is assigned to {targetType}"
+    prompt = "The issue is assigned to {No One/Team/Member}"
+
+    form_fields = {"targetType": {"type": "assignee", "choices": CHOICES}}
+
+    def passes(self, event, state):
+        targetType = self.get_option("targetType")
+
+        issue_assignees = event.group.assignee_set
```

Comment:
Wondering if we might need some caching here... This could cause a lot of db queries in a high event org. It might be safest to cache the set of `assignees` for each group for a minute or so, whatever our default cache time is. We don't have any other conditions that make a query like this, so likely is the safest option.